pipeline.groovy

```diff
@@ -191,17 +201,16 @@ def testDownstreamProject (name) {
                 case 'BlackEdder/ggplotd':
                     // workaround https://github.com/BlackEdder/ggplotd/issues/34
                     sh 'sed -i \'s|auto seed = unpredictableSeed|auto seed = 54321|\' source/ggplotd/example.d'
-                    sh 'dub test'
+                    test_travis_yaml()
+                    break;
+
+                case 'rejectedsoftware/diet-ng':
+                    sh 'sed -i \'s|dependency "vibe-d".*|dependency "vibe-d" version="0.7.31-beta.2"|\' examples/htmlserver/dub.sdl'
```

Comment:
This might break again when Sönke adds a new test/example - how about  `**/*.sdl` ?